src/App.css

```diff
@@ -1 +1,43 @@
-
+.app {
+  margin: 0 auto;
+  width: 100%;
+  text-align: center;
+}
+.match-content {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-area: 5 / 1 / 6 / 2;
+  grid-gap: 1rem;
+  margin: 0 auto;
+  justify-content: center;
+  align-items: center;
+  width: 40%;
+  height: 40%;
+  background-color: #f5f5f5;
+}
+h1,
+h3 {
+  text-align: center;
+  font-size: 2em;
+  margin: 0;
+  padding: 0;
+}
+.items {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2em;
+  border: 1px solid #000;
+  border-radius: 5px;
+  padding: 10px;
+  margin: 0 auto;
+  width: 100%;
+  height: 100%;
+  text-transform: capitalize;
+}
+img {
```

Comment:
Let's be more specific here, use a class name. Generally don't use element selectors like `img` or `h1`.